src/Service/FormInputService.php

```diff
@@ -0,0 +1,192 @@
+class FormInputService
+{
+
+    /**
+     * @var EntityManagerInterface
+     */
+    private EntityManagerInterface $entityManager;
+
+    /**
+     * @var SynchronizationService
+     */
+    private SynchronizationService $syncService;
+
+    /**
+     * @var CallService
+     */
+    private CallService $callService;
+
+    /**
+     * @var CacheService
+     */
+    private CacheService $cacheService;
+
+    /**
+     * @var GithubApiService
+     */
+    private GithubApiService $githubApiService;
+
+    /**
+     * @var GithubPubliccodeService
+     */
+    private GithubPubliccodeService $publiccodeService;
+
+    /**
+     * @var ImportResourcesService
+     */
+    private ImportResourcesService $importResourcesService;
+
+    /**
+     * @var EnrichPubliccodeFromGithubUrlService
+     */
+    private EnrichPubliccodeFromGithubUrlService $enrichPubliccode;
+
+    /**
+     * @var FindGithubRepositoryThroughOrganizationService
+     */
+    private FindGithubRepositoryThroughOrganizationService $organizationService;
+
+    /**
+     * @var FindOrganizationThroughRepositoriesService
+     */
+    private FindOrganizationThroughRepositoriesService $findOrganization;
+
+    /**
+     * @var GatewayResourceService
+     */
+    private GatewayResourceService $resourceService;
+
+    /**
+     * @var LoggerInterface
+     */
+    private LoggerInterface $pluginLogger;
+
+    /**
+     * @var array
+     */
+    private array $configuration;
+
+    /**
+     * @var array
+     */
+    private array $data;
+
+
+    /**
+     * @param EntityManagerInterface                         $entityManager          The Entity Manager Interface.
+     * @param SynchronizationService                         $syncService            The Synchronization Service.
+     * @param CallService                                    $callService            The Call Service.
+     * @param CacheService                                   $cacheService           The Cache Service.
+     * @param GithubApiService                               $githubApiService       The Github Api Service.
+     * @param GithubPubliccodeService                        $publiccodeService      The Github Publiccode Service.
+     * @param ImportResourcesService                         $importResourcesService The Import Resources Service.
+     * @param EnrichPubliccodeFromGithubUrlService           $enrichPubliccode       The Enrich Publiccode From Github Url Service.
+     * @param FindGithubRepositoryThroughOrganizationService $organizationService    The find github repository through organization service.
+     * @param GatewayResourceService                         $resourceService        The Gateway Resource Service.
+     * @param LoggerInterface                                $pluginLogger           The plugin version of the logger interface
+     * @param FindOrganizationThroughRepositoriesService     $findOrganization       The find organization through repositories service.
+     */
```

Comment:
Do we need to import and construct all of these services?

Reply:
No I need to cleanup the imports of the services, I will do that in the next pr